src/components/Person/Person.jsx

```diff
@@ -1 +1,13 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export function Person(props) {
+  const { name, age, sex, isMarried, partnerName } = props.person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && (<p className="Person__age">{`I am ${age}`}</p>)}
+      <p className="Person__partner">{isMarried ? `${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}` : 'I am not married'}</p>
```

Comment:
```suggestion
      <p className="Person__partner">
        {isMarried
          ? `${partnerName} is my ${partner'}`
          : 'I am not married'}
        </p>
```

Try to avoid nesting ternary:

You can create separate variable partner and move this part `sex === 'm' ? 'wife' : 'husband'` into it instead